### DISCUSSION TITLE
Suggestion: add classify( ) method

### DISCUSSION THREAD

**Discussion by 2MinuteWarning** [fcakyon/yolov5-pip#158] [Q&A] [answered]
First off, I *love* this package and use it a lot.  Thank you so much for it.

Now that Yolov5 has added classification support it would be great if there was a classify( ) inference method callable from Python in the same way there is for object detection.

Is this planned for the future?

**Comment by fcakyon** [comment]
@2MinuteWarning i am currently very busy with my PhD but yes this is a planned feature with no ETA 👍

**Comment by fcakyon** [comment, marked as answer]
It is currently supported.
